### PR TITLE
:wrench: Update Play Services framework tag to work with phonegap-plugin-push

### PR DESCRIPTION
**Pull request by macdonst**
This change works around an incompatibility with Play Services and phonegap-plugin-push. Folks can still over-ride the Play Services version using a preference/variable but if you leave it as + that causes issues with the Google Services Gradle Plugin.

Sorry about the formatting changes. A combination of Prettier, eslint, VS Code auto styling.

### CODE REVIEW

**Comment by ratson** [comment]
@macdonst Thank you for the PR.
This is repo is obsoleted.
Would you send your PR to https://github.com/rehy/cordova-admob-sdk?
Please also add a section in README about how the customization could be done.

File: plugin.xml [hunk 1]
```diff
@@ -1,83 +1,72 @@
...
     <name>Google Mobile Ads SDK for Cordova</name>
     <description>Add Google Mobile Ads SDK to Cordova project, as dependency for other plugins</description>
     <author>Google</author>
     <keywords>admob,google,ad</keywords>
     <repo>https://github.com/sunnycupertino/cordova-admob-sdklibs.gitt</repo>
     <issue>https://github.com/sunnycupertino/cordova-admob-sdklibs/issues</issue>
-
     <engines>
         <engine name="cordova" version=">=3.0" />
+        <engine name="cordova-android" version=">=6.3.0"/>
```
Comment (ratson) [inline plugin.xml:13]: @macdonst Is this a requirement for the change?

Comment (macdonst) [reply plugin.xml:13]: @ratson yes, without it the environment variable won't be replaced correctly.

**Comment by macdonst** [comment]
@ratson I will make the necessary updates in the correct repo.

**Pull request closed without merging**